codes/write_dirs_files_for_gamess_qcxms.py

```diff
@@ -277,13 +277,13 @@ def write_gamess_input(multiplicity: int, mol: object, molname: str, mol_input_p
             write_gamess_input(multiplicity, mol, molname, mol_input_path)
 
         spectrum_input_path = spectra_dir / ("qcxms" + ".in")
-        qcxms_template = load_template("templates", "qcxms_input_template.in")
+        qcxms_template = load_template("qcxms_input_template.in")
         if not Path(spectrum_input_path).exists():
-            qcxms_params = read_parameters(args.params_filename, ["QC_Program", "QC_Level", "ntraj", "tmax", "tinit", "ieeatm"])
+            qcxms_params = read_parameters(args.params_filename, ["QC_Program", "QC_Level", "ntraj", "tmax", "tinit"])
             write_from_template(parameters=qcxms_params, template=qcxms_template, file=spectrum_input_path)
 
         mol_pbs_path = mol_dir / (inchikey + ".pbs")
-        pbs_template = load_template("templates", "optimization_gamess_template.pbs")
+        pbs_template = load_template("optimization_gamess_template.pbs")
```

Comment:
You can re-use the PBS_TEMPLATE variable here

Reply:
I use one variable now `template` [2ffd3ef](https://github.com/RECETOX/ei_spectra_predictions/pull/43/commits/2ffd3ef950cbfe0514a2c25a138e8da90901d371)

Reply:
@hechth, If you agree. We can merge this PR?

Reply:
this is not what I meant - now you are overwriting the variable, not sure if this will work.